test/embedded_manifold.jl

```diff
@@ -40,11 +62,11 @@ struct NotImplementedEmbeddedManifold3 <: AbstractEmbeddedManifold{ℝ,DefaultEm
         M = PlaneManifold()
         @test repr(M) == "PlaneManifold()"
         @test ManifoldsBase.default_decorator_dispatch(M) === Val{false}()
-        @test get_embedding(M) == ManifoldsBase.DefaultManifold(3)
+        @test get_embedding(M) == ManifoldsBase.DefaultManifold(1, 3)
         # Check fallbacks to check embed->check_manifoldpoint Defaults
-        @test_throws DomainError is_manifold_point(M, [1, 0], true)
-        @test_throws DomainError is_tangent_vector(M, [1, 0], [1, 0, 0], true)
-        @test_throws DomainError is_tangent_vector(M, [1, 0, 0], [0, 0], true)
+        @test is_manifold_point(M, [1, 0], true)
```

Comment:
Why are these now working? Especially the tangent vector should not work...

Reply:
That's because embedding itself can "fix" points now -- it doesn't check if the representation before embedding is correct, just copies what is needed. New methods of `allocate_result` ensure that an array of the right size is filled.

Reply:
Ah, ok.